### PR TITLE
First pass at saving map config

### PR DESCRIPTION
**Pull request by bartvde**
see also #153

### CODE REVIEW

File: examples/full-example.html [hunk 1]
```diff
@@ -107,6 +107,7 @@
                     <li ng-class="selected.pins && 'active'"><a ng-click="selected = {pins:true}">Pins</a></li>
                     <li ng-class="selected.preview && 'active'"><a ng-click="selected = {preview:true}">Preview</a></li>
                 </ul>
+                <button style="position: absolute; top: 2px; right: 2px;" ng-click="saveMap()" class="btn btn-primary">Save Map</button>
```
Comment (ischneider) [inline examples/full-example.html]: does the bootstrap `pull-right` class work here?

**Comment by ischneider** [comment]
looks like a good start

File: examples/full-example.html (hunk 1 above)
Comment (bartvde) [inline examples/full-example.html]: it does but it will be in a second "row"

**Comment by ischneider** [comment]
rebased and fixed conflicts

**Pull request closed without merging**